Extensions/CMake/SlicerBlockBuildPackageAndUploadExtension.cmake

```diff
@@ -153,6 +153,7 @@ MIDAS_PACKAGE_URL:STRING=${MIDAS_PACKAGE_URL}
 MIDAS_PACKAGE_EMAIL:STRING=${MIDAS_PACKAGE_EMAIL}
 MIDAS_PACKAGE_API_KEY:STRING=${MIDAS_PACKAGE_API_KEY}
 EXTENSION_DEPENDS:STRING=${EXTENSION_DEPENDS}
+EXTENSION_TAGS:STRING=${EXTENSION_TAGS}
```

Comment:
Everywhere where we talk about "tags", it should be updated to the chosen final name.

Reply:
Perhaps EXTENSION_BUILD_REQUIREMENTS?

Reply:
EXTENSION_BUILD_REQUIREMENTS is good if we just want to use this for managing special build requirements. However, I think we could address the future requirement of custom build groups by calling this EXTENSION_BUILD_GROUPS (and in the s4ext file buildGroups).

We can make up build groups based on special library requirements but also based on other considerations (we could define groups of extensions that don't depend on extensions in other groups, and build each group in parallel on a separate factory machine to distribute the load).

Reply:
I like it, a site could have multiple machines, each taking ~1/Xth the extensions to build.

Reply:
Just a thought, one benefit of keeping it generic like "tags" is that you could have tags like build_group_XYZ, build_group_ABC, but also tags like build_req_cuda, or anything really... 

Pros are flexibility, cons are need for documentation. Could have a wiki page with known tags, and a link to that page everywhere they're used?

Reply:
@lassoan @jcfr 
Reminder of outstanding discussion:

Keep tag system generic (use descriptive tags) or focus tag concept to be specific to build system (rename to EXTENSION_BUILD_GROUPS)?